users/hvp/rules.mk

```
@@ -0,0 +1,2 @@
+SRC += hvp.c
+SRC += tap_dances.c
```

You can actually get it to include the file only if the feature is enabled:
```suggestion
ifeq ($(strip $(TAP_DANCE_ENABLE)), yes)
    SRC += tap_dances.c
endif```